kiln/devnets/goerlishadow-4.vars

```diff
@@ -1,21 +1,28 @@
 DEVNET_NAME=goerlishadow-4
+CONFIG_GIT_DIR=goerli-shadow-fork-4
+NETWORK_ID=5
+MERGE_TTD=9895202
+
+JWT_SECRET="0xdc6457099f127cf0bac78de8b297df04951281909db4f58b43def7c7151e765d"
 
 GETH_IMAGE=parithoshj/geth:master
 NETHERMIND_IMAGE=nethermindeth/nethermind:kiln_shadowfork
 ETHEREUMJS_IMAGE=g11tech/ethereumjs:kiln
-LODESTAR_IMAGE=chainsafe/lodestar:next
-CONFIG_GIT_DIR=goerli-shadow-fork-4
+BESU_IMAGE=hyperledger/besu:22.4.0-RC2-SNAPSHOT
```

Comment:
`22.4.0-RC2-SNAPSHOT`, is currently the latest build, but `develop` might be a more future-proof tag here